mantle/platform/platform.go

```diff
@@ -476,9 +476,25 @@ func CheckMachine(ctx context.Context, m Machine) error {
 		return fmt.Errorf("not a supported instance: %v", string(out))
 	}
 
+	// check systemd version on host to see if we can use `busctl --json=short`
+	var systemdVer string
+	var failedUnitsCmd string
+	minSystemdVer := "240"
+	out, stderr, err = m.SSH("rpm -q --queryformat='%{VERSION}\n' systemd")
+	if err != nil {
+		return fmt.Errorf("failed to query systemd RPM for version: %s: %v: %s", out, err, stderr)
+	}
+	systemdVer = string(out[:])
+
+	if systemdVer >= minSystemdVer {
```

Comment:
just noting here that this is a string comparison, which works, but it confused me for a minute because the version is a number (represented as a string).

Reply:
Could we convert everything to integers instead?